src/object/schema_system_catalog_install.cpp

```diff
@@ -970,7 +970,7 @@ namespace cubschema
 		   // columns
     {
       {"db_name", format_varchar (255)},
-      {"db_creation_time", "datetime"},
+      {"creation_time", "datetime"},
```

Comment:
```suggestion
      {"db_creation_time", "datetime"},
```
db_ha_apply_info 시스템 카탈로그의 필드명을 임의로 변경하면 안됩니다.

Reply:
원복하였습니다.